### ISSUE TITLE
QT wallet allows same scape address as source address; paycoind does not

### ISSUE THREAD

**Issue by mdpfeiffer** [PaycoinFoundation/paycoin#364]
QT wallet allows same scape address as staking address.

By contrast, paycoind returns error message:

error: {"code":-1,"message":"Cannot set scrape address to the same as staking address."}

Not sure which one is right, or if the discrepancy could cause problems.

**Comment by IngCr3at1on** [comment]
Yeah it shouldn't allow the same address, though clearly I failed to handle that on the QT side....
